internal/cli/flags.go

```diff
@@ -62,8 +64,16 @@ func addDevServerFlags(cmd *cobra.Command) {
 	cmd.Flags().Var(&webModeFlag, "web-mode", "Values: local, prod. Controls whether to use prod assets or a local dev server. (If flag not specified: if Tilt was built from source, it will use a local asset server; otherwise, prod assets.)")
 }
 
+func addNamespaceFlag(cmd *cobra.Command) {
+	cmd.Flags().StringVar(&namespaceFlag, "namespace", defaultNamespace, "Namespace for the resources Tilt brings up.")
```

Comment:
This is tricky to convey concisely, but I think it'd be good to mention that it's a default, e.g. "Default namespace for Kubernetes resources (overrides default namespace from active context in kubeconfig)"